src/main-thread/worker.ts

```diff
@@ -80,7 +83,11 @@ export class WorkerContext {
       }).call(self);
       ${authorScript}
       //# sourceURL=${encodeURI(config.authorURL)}`;
-    this[TransferrableKeys.worker] = new Worker(URL.createObjectURL(new Blob([code])));
+    if (!config.sandbox) {
+      this[TransferrableKeys.worker] = new Worker(URL.createObjectURL(new Blob([code])));
+    } else if (IS_AMP && config.sandbox){
```

Comment:
By this point in the conditional, `!config.sandbox` was false so `config.sandbox` should be truthy. 
```suggestion
    } else if (IS_AMP) {
```

Reply:
Thats true! I think it is less obvious to a reader though which is why I intentionally left this in. Don't feel too strongly, I'll update